zeus_arm:
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50

  joint_1_position_controller:
    type: effort_controllers/JointPositionController
    joint: joint_1
    pid: {p: 27.0, i: 6.0, d: 17.0}
  joint_2_position_controller:
    type: effort_controllers/JointPositionController
    joint: joint_2
    pid: {p: 300.0, i: 10.0, d: 1.0}
  joint_3_position_controller:
    type: effort_controllers/JointPositionController
    joint: joint_3
    pid: {p: 150.0, i: 10.0, d: 0.0}
  joint_4_position_controller:
    type: effort_controllers/JointPositionController
    joint: joint_4
    pid: {p: 150.0, i: 5.0, d: 0.0}
  joint_5_position_controller:
    type: effort_controllers/JointPositionController
    joint: joint_5
    pid: {p: 10.0, i: 0.0, d: 0.0}

